src/initial-setup/com_redhat_subscription_manager/ks/rhsm_ks.py

```diff
@@ -147,21 +147,35 @@ def __init__(self, name):
     def __str__(self):
         return "%%addon %s %s\n%s%%end\n" % (self.name, self.header_args, self.content)
 
-    def setup(self, storage, ksdata, instclass, payload):
+    def setup(self, *args, **kwargs):
         """Make the changes to the install system.
 
            This method is called before the installation
            is started and directly from spokes. It must be possible
            to call it multiple times without breaking the environment."""
+        # we use the signature we do because anaconda before commit
+        # 78fd1e82 passes (storage, ksdata, instClass, payload)
+        # but anaconda after 78fd1e82 passes only
+        # (storage, ksdata, payload). If this method *did*
+        # anything we would have to figure out which args we'd got and
+        # handle them appropriately, but since it does nothing, we
+        # don't have to bother with that.
```

Comment:
I'm not native English speaker, but please, can you change the comment to use little bit better English grammar?